roles/manage-aws-infra/tasks/manage-cf-stack.yml

```diff
@@ -0,0 +1,20 @@
+---
+- name: Ensure the underlaying infrastructure is {{ action }}
+  cloudformation:
+    stack_name: "{{ env_id }}"
+    state: "{{ action }}"
+    region: "{{ aws_region }}"
+    template: "{{ cftemplate.dest }}"
+    template_parameters:
+      VPCCIDR: "{{ VPCCIDR }}"
+      PrivateSubnet1CIDR: "{{ PrivateSubnet1CIDR }}"
+      PrivateSubnet2CIDR: "{{ PrivateSubnet2CIDR }}"
+      PrivateSubnet3CIDR: "{{ PrivateSubnet3CIDR }}"
+      Image: "{{ aws_image_name }}"
```

Comment:
This should be the image specified in the `cloud_infrastructure`

Reply:
Done